src/platform/plugins/shared/esql/public/triggers/index.ts

```diff
@@ -11,7 +11,13 @@ export {
   updateESQLQueryTrigger,
```

Comment:
I would recommend deleting this index file. Index files make it very easy to leak a bunch of stuff into page load bundle. Instead, just have plugin just directly import resources as needed.

Reply:
Same